app/controllers/hub/organizations_controller.rb

```diff
@@ -86,8 +92,10 @@ def activate_all_role_param
       params.require(:role_type)
     end
 
-    def organization_params
-      params.require(:organization).permit(:name, :coalition_id, :timezone, :capacity_limit, :allows_greeters, source_parameters_attributes: [:_destroy, :id, :code])
+    def organization_form_params
+      params.require(:hub_organization_form).permit(
+        :name, :is_independent, :states, :coalition_id, :timezone, :capacity_limit, :allows_greeters, source_parameters_attributes: [:_destroy, :id, :code]
```

Comment:
maybe we can remove `:id` from `source_parameters_attributes` and it would still work